lib/modules/datasource/hex/index.ts

```diff
@@ -18,6 +18,13 @@ export class HexDatasource extends Datasource {
 
   override readonly defaultVersioning = hexVersioning.id;
 
+  override readonly releaseTimestampSupport = true;
+  override readonly releaseTimestampNote =
+    'The relase timestamp is determined the `inserted_at` field in the results.';
+  override readonly sourceUrlSupport = 'package';
+  override readonly sourceUrlNote =
+    'The source url from the `Github` field present in the results.';
```

Comment:
```suggestion
    'The source URL is determined from the `Github` field present in the results.';
```

I think this is the correct pattern?